meilisearch-http/src/main.rs

```diff
@@ -94,37 +93,41 @@ pub fn print_launch_resume(opt: &Opt, data: &Data) {
 888       888  "Y8888  888 888 888  "Y8888P"   "Y8888  "Y888888 888     "Y8888P 888  888
 "#;
 
-    info!("{}", ascii_name);
+    println!("{}", ascii_name);
 
-    info!("Database path: {:?}", opt.db_path);
-    info!("Start server on: {:?}", opt.http_addr);
-    info!("Environment: {:?}", opt.env);
-    info!("Commit SHA: {:?}", env!("VERGEN_SHA").to_string());
-    info!(
-        "Build date: {:?}",
+    println!("Database path:\t\t{:?}", opt.db_path);
+    println!("Start server on:\t{:?}", opt.http_addr);
+    println!("Environment:\t\t{:?}", opt.env);
+    println!("Commit SHA:\t\t{:?}", env!("VERGEN_SHA").to_string());
+    println!(
+        "Build date:\t\t{:?}",
         env!("VERGEN_BUILD_TIMESTAMP").to_string()
     );
-    info!(
-        "Package version: {:?}",
+    println!(
+        "Package version:\t{:?}",
         env!("CARGO_PKG_VERSION").to_string()
     );
 
+    println!();
+
     if let Some(master_key) = &data.api_keys.master {
-        info!("Master Key: {:?}", master_key);
+        println!("Master Key:\t{:?}", master_key);
 
         if let Some(private_key) = &data.api_keys.private {
-            info!("Private Key: {:?}", private_key);
+            println!("Private Key:\t{:?}", private_key);
         }
 
         if let Some(public_key) = &data.api_keys.public {
-            info!("Public Key: {:?}", public_key);
+            println!("Public Key:\t{:?}", public_key);
         }
     } else {
-        info!("No master key found; The server will have no securities.\
+        println!("No master key found; The server will have no securities.\
```

Comment:
```suggestion
        println!("No master key found; The server accepts unidentified requests. \
```
(don't forget the space at the end)